.github/workflows/string-extraction-localization.yml

```
@@ -0,0 +1,152 @@
+      # 7. Validation
+      - name: Validate Changes
+        if: steps.extraction.outputs.strings-found == 'true'
+        run: |
+          python .github/scripts/validate-changes.py \
+            --base-path ./presentation/compose/src/commonMain \
+            --extracted-strings /tmp/extracted-strings/processed-strings.json
+
+      # 8. Create Pull Request
+      - name: Create Localization PR
+        if: steps.extraction.outputs.strings-found == 'true'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.10)

[error] the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)